parachain/src/service.rs

```diff
@@ -230,28 +229,21 @@ where
             registry,
         )
     } else {
+        let slot_duration = cumulus_client_consensus_aura::slot_duration(&*client)?;
```

Comment:
Did you check that this returns the correct duration? We used to hardcode to 12000 in 1.20.1

Reply:
Tested on Kintsugi and it was still reading 6000, I had assumed it would use the latest block but that was not the case. Hard-coding to 12000 for now, we can investigate a better solution when we have more time but this change is required for the next upgrade.

Reply:
> this change is required for the next upgrade.

How so? Source?

Reply:
The collators will fail to produce blocks if they read 6000 as the `slot_duration`, we already have this change in the patched version: https://github.com/interlay/interbtc/commit/aca2f3e595b3176bc6673ed7feef4c5d88e3e1d9

Reply:
Ah yea, I misread your comment - I agree the 12000ms hardcoding is required. I thought you were saying your alternative approach was required